src/main/java/com/teragrep/k8s_01/K8SConsumer.java

```diff
@@ -132,6 +132,21 @@ public void accept(FileRecord record) {
                 );
             }
             Instant instant = Instant.parse(log.getTimestamp());
```

Comment:
this does not fix the npe, npe is caused by Instant.parse(null) not by "instant" being null.